src/test/java/api/loans/scenarios/ChangeDueDateTests.java

```diff
@@ -287,4 +292,28 @@ public void itemIsStillDeclaredLostWhenLoanDueDateIsChanged() {
     assertThat("Should not contain check out service point summary",
       loanInStorage.containsKey("checkoutServicePoint"), is(false));
   }
+
+  @Test
+  public void cannotChangeDueDateForDeclaredLostItem() {
+    final IndividualResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final IndividualResource loan = loansFixture.checkOutByBarcode(smallAngryPlanet);
+    final JsonObject loanJson = loan.getJson();
+
+    loansFixture.declareItemLost(loanJson);
+
+    Response fetchedLoan = loansClient.getById(loan.getId());
+    JsonObject loanToChange = fetchedLoan.getJson().copy();
+    DateTime dueDate = DateTime.parse(loanToChange.getString("dueDate"));
+    DateTime newDueDate = dueDate.plus(Period.days(14));
+    write(loanToChange, "action", "dueDateChange");
+    write(loanToChange, "dueDate", newDueDate);
+
+    Response response = loansFixture.attemptToReplaceLoan(loan.getId(), loanToChange);
+
+    assertThat("The status code is UNPROCESSABLE_ENTITY", response.getStatusCode(),
```

Comment:
Personally, I think the comment is redundant as it is a restatement of the assertion. Could the `ResponseStatusCodeMatcher` be used instead?

Reply:
Changed